client/src/components/actual-achievement-item/actual-achievement-item.css

```diff
@@ -0,0 +1,27 @@
+.actual-achievement-item {
+  margin: 10px;
+  text-align: center;
+  width: 150px;
+}
+
+.actual-achievement-item__button {
+  background-color: orange;
+  border: none;
+  width: 65px;
+  height: 65px;
+  border-radius: 6px;
+}
+
+.actual-achievement-item__button--done {
+  background-color: green;
```

Comment:
Actually it's better to use hex codes instead of color names, because names may give the different shades in the different browsers.